### PR TITLE
Feature/offpay 463

### PR DESCRIPTION
**Pull request by miloszm**

### CODE REVIEW

File: test/resources/schema/1.0.0-final/off-payroll-request-schema.json [hunk 1]
```diff
@@ -75,121 +53,96 @@
...
-        },
-        "financialRiskB": {
-          "type": "object",
-          "properties": {
-            "provideVehicle": {
+            "incomeCalendarPeriods": {
               "$ref": "#/definitions/binaryAnswer"
             },
-            "engagerPayVehicleExpense": {
+            "incomeFixed": {
```
Comment (habeeb01) [inline test/resources/schema/1.0.0-final/off-payroll-request-schema.json]: workerMainIncome and paidForSubstandardWork should be top level attributes

File: test/resources/test-scenarios/1.0.0-final/matrix/scenario-decision-unknown.csv [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+control             ,   workerDecidingHowWorkIsDone               ,  workerAgreeWithOthers
+control             ,   whenWorkHasToBeDone                       ,  scheduleDecidedForWorker
+control             ,   workerDecideWhere                         ,  workerAgreeWithOthers
+financialRisk       ,   workerProvidedMaterials                   ,  No
+financialRisk       ,   workerProvidedEquipment                   ,  No
+financialRisk       ,   workerUsedVehicle                         ,  No
+financialRisk       ,   workerHadOtherExpenses                    ,  No
+financialRisk       ,   expensesAreNotRelevantForRole             ,  Yes
+financialRisk       ,   incomeCalendarPeriods                     ,  No
+financialRisk       ,   incomeFixed                               ,  Yes
```
Comment (habeeb01) [inline test/resources/test-scenarios/1.0.0-final/matrix/scenario-decision-unknown.csv]: this is related to the above comment (workerMainIncome and paidForSubstandardWork should be top level)

File: test/uk/gov/hmrc/decisionservice/services/ComplianceTestCasesSpec.scala [hunk 3]
```diff
@@ -51,9 +49,9 @@ class ComplianceTestCasesSpec extends UnitSpec {
         maybeDecision.isValid shouldBe true
         maybeDecision.map { decision =>
           decision.value shouldBe "Not Matched"
-          val maybeBusinessStructureScore = Score.create(decision.facts).get("businessStructure")
+          val maybeBusinessStructureScore = Score.create(decision.facts, Versions.VERSION1).get("businessStructure")
           maybeBusinessStructureScore.isDefined shouldBe true
-          maybeBusinessStructureScore.map{businessStructureScore =>
+          maybeBusinessStructureScore.map { businessStructureScore =>
```
Comment (habeeb01) [inline test/uk/gov/hmrc/decisionservice/services/ComplianceTestCasesSpec.scala]: is this still a valid test seeing as businessStructure is gone?

File: app/uk/gov/hmrc/decisionservice/model/api/ApiModel.scala [hunk 4]
```diff
@@ -30,11 +30,20 @@ case class Score( score:Map[String,String])
...
   implicit val scoreFormat: Format[Score] = Json.format[Score]
-  val elements = List("control", "financialRiskA", "financialRiskB", "partAndParcel", "businessStructure", "personalService")
-  def create(facts:Map[String,CarryOver]):Map[String,String] =
-    facts.toList.collect { case (a,co) if (Score.elements.contains(a)) => (a,formatValue(co.value))}.toMap
-  def createRaw(m:Map[String,String]) = m
-  def formatValue(value:String) = value match {
+  def elements(version:String):List[String] = {
+    val versionToElements = Map(
+      "1.0.1-beta" -> List("control", "financialRiskA", "financialRiskB", "partAndParcel", "businessStructure", "personalService"),
+      "1.0.0-final" -> List("control", "financialRisk", "partAndParcel", "businessStructure", "personalService")
```
Comment (peteanning) [inline app/uk/gov/hmrc/decisionservice/model/api/ApiModel.scala]: Business Structure has gone

File: test/uk/gov/hmrc/decisionservice/services/ComplianceTestCasesSpec.scala (hunk 3 above)
Comment (miloszm) [inline test/uk/gov/hmrc/decisionservice/services/ComplianceTestCasesSpec.scala]: Habeeb - business structure is not gone for version 1.0.1-beta, and this test is for this version.

**Pull request merged**